packages/amplify-auth-sdk/package.json

```diff
@@ -22,19 +22,19 @@
     "@axway/amplify-request": "^0.4.0",
     "accepts": "^1.3.5",
     "fs-extra": "^7.0.1",
-    "jws": "^3.2.1",
-    "opn": "^5.4.0",
+    "jws": "^3.2.2",
+    "opn": "^6.0.0",
```

Comment:
Deprecated in place of `open` shall we move to that?

Reply:
Yes, absolutely. Stay tuned.